src/states_screens/race_gui_overworld.cpp

```diff
@@ -443,6 +443,17 @@ void RaceGUIOverworld::drawGlobalMiniMap()
         const ChallengeData* challenge = unlock_manager->getChallengeData(challenges[n].m_challenge_id);
         const unsigned int val = challenge->getNumTrophies();
         bool unlocked = (PlayerManager::getCurrentPlayer()->getPoints() >= val);
+        if (challenges[n].m_challenge_id == "fortmagma") {
```

Comment:
Bracket goes on a newline.

Also, I'm not a huge fan of hardcoding "fortmagma" in the code :( Though the alternatives I can think of require a lot more code and may not be worth it... Maybe someone has an idea

Reply:
I'm not really a fan of it either... but there is no marker for the final track, and adding it would require adding stuff which may/may not go along the rest of the code's architecture...